intelmq/lib/splitreports.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
All this code should move to https://github.com/certtools/intelmq/blob/master/intelmq/lib/bot.py#L576